modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/summary/Summary.js

```diff
@@ -78,19 +78,17 @@ function SummaryItem(props) {
 
 	let ItemVariant;
 
-	switch (style) {
-		case 'big':
-			ItemVariant = SummaryItemBigVariant;
-			break;
-		case 'divider':
-			ItemVariant = SummaryItemDividerVariant;
-			break;
-		case 'danger':
-			ItemVariant = SummaryItemDangerVariant;
-			break;
-		default:
-			ItemVariant = SummaryItemBase;
-			break;
+	if (style === 'big') {
+		ItemVariant = SummaryItemBigVariant;
+	}
+	else if (style === 'danger') {
+		ItemVariant = SummaryItemDangerVariant;
+	}
+	else if (style === 'divider') {
+		ItemVariant = SummaryItemDividerVariant;
+	}
+	else {
```

Comment:
You could also remove the final `else` and simply initialize `ItemVariant` as follows:

```javascript
let ItemVariant = SummaryItemBase;
```

to be consistent with the approach you used in all of the other files.